CorsixTH/Lua/world.lua

```diff
@@ -248,6 +248,15 @@ function World:initLevel(app, avail_rooms)
       end
     end
   end
+  -- Apply level-defined pricing to diseases on new game:
+  if level_config and level_config.expertise and #self.available_diseases > 0 then
+    for _, dis in ipairs(self.available_diseases) do
+      local ex = dis.expertise_id and level_config.expertise[dis.expertise_id]
+      if ex and ex.StartPrice ~= nil then
```

Comment:
Based on your commit message you may not be aware, 0 is truthy in lua (unlike javascript) so the ~= nil isn't needed.